agents/unified/mcp_a2a_mojo_integration.py

```
@@ -387,66 +411,81 @@ async def demonstrate_unified_architecture():
         "symbol": "AAPL",
         "quantity": 10000,
         "type": "MARKET",
-        "exchange": "NASDAQ"
+        "exchange": "NASDAQ",
     }
-    
+
     result = await trader.execute_market_order(order)
-    
+
     print(f"✓ Order executed")
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)